### PR TITLE
[Finishes #187355052] Users should be presented with a reset password

### PR DESCRIPTION
**Pull request by Christian-Ishimwe**
### What does this PR do?
This PR, allows a user or seller to be able to reset their password

### How should it be tested

1. Clone the repo, navigate to the branch `git checkout  fx-reset-password`
2. After, install all dependencies by `npm install`
3. check `.env.example` and set the required variables
4. start the applicatiin by: `npm start`
5. after that, navigate to `/login`
6. continue the process, click on `forgot password? `

### Required screenshot

Screenshot 2024-06-22 at 12 34 19
Screenshot 2024-06-22 at 12 34 41
Screenshot 2024-06-22 at 12 34 56
Screenshot 2024-06-22 at 12 36 18
Screenshot 2024-06-22 at 12 36 31
Screenshot 2024-06-22 at 12 46 38

### CODE REVIEW

**Review by shema-surge — APPROVED** [review]

**Review by Celestin4 — APPROVED** [review]

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
The reset route is broken
Screenshot 2024-06-22 at 22 34 49

File: .DS_Store
Comment (UmuhireJessie) [inline .DS_Store:1]: Remove this file and add it in `.gitignore` file

File: __tests__/.DS_Store
Comment (UmuhireJessie) [inline __tests__/.DS_Store:1]: Remove this file and add it in `.gitignore`

File: src/views/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Link } from 'react-router-dom';
+const Login: React.FC = () => (
+  <div>
+    <h1>Login Page</h1>
+    <p>Please login to continue.</p>
+
+    <p>You dont have account, please clieck here to   <Link to='/signup'>Signup</Link></p>
+
+   
+    <Link to={'/reset'}>Forgot password?</Link>
```
Comment (UmuhireJessie) [inline src/views/Login.tsx:11]: Is there a perfect way to show this instead of having it as a link? You can think of italics, hovering effect,...

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Good work @Christian-Ishimwe , please note:
1. Can you add all the environment variables for the testing purpose
Screenshot 2024-07-02 at 09 04 33

2. Can you improve the design of this link 
Screenshot 2024-07-02 at 09 05 54

**Review by pacifiquemboni — APPROVED** [review]

**Review by lilika67 — APPROVED** [review]

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**